src/reconstruction/CoreModel.jl

```diff
@@ -1,3 +1,44 @@
+"""
+    $(TYPEDSIGNATURES)
+
+Add `rxns` to `model` efficiently. Note, the model must already contain the metabolites used by
```

Comment:
remove "Note, " -- the model simply has to contain the metabolites. :D

(same below)